infrastructure/protocol-upgrade/src/hyperchain-upgrade.ts

```diff
@@ -58,6 +61,21 @@ async function hyperchainUpgrade1() {
     );
 }
 
+async function insertAddresses() {
```

Comment:
why do we need it? we will have the new facets deployed and they would be written to the facets.json

Reply:
I am using a different deployment script, because we also need to deploy the STM, Bridgehub, and other contracts, and we need to set the initial diamond cut in the STM. Doing this seemed easier.